[2023-02-02T05:28:36.630+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-29T00:00:00+00:00 [queued]>
[2023-02-02T05:28:36.638+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-29T00:00:00+00:00 [queued]>
[2023-02-02T05:28:36.638+0530] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T05:28:36.638+0530] {taskinstance.py:1284} INFO - Starting attempt 2 of 4
[2023-02-02T05:28:36.638+0530] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T05:28:36.654+0530] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract> on 2023-01-29 00:00:00+00:00
[2023-02-02T05:28:36.662+0530] {standard_task_runner.py:55} INFO - Started process 6152 to run task
[2023-02-02T05:28:36.665+0530] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'NSE_Open_Interest', 'extract', 'scheduled__2023-01-29T00:00:00+00:00', '--job-id', '3578', '--raw', '--subdir', 'DAGS_FOLDER/NSE_Open_Interest.py', '--cfg-path', '/tmp/tmpwv0otd9m']
[2023-02-02T05:28:36.666+0530] {standard_task_runner.py:83} INFO - Job 3578: Subtask extract
[2023-02-02T05:28:36.715+0530] {task_command.py:389} INFO - Running <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-29T00:00:00+00:00 [running]> on host DESKTOP-SQIOQPD.
[2023-02-02T05:28:36.781+0530] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=nikheleshbhattacharyya@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=NSE_Open_Interest
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2023-01-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-29T00:00:00+00:00
[2023-02-02T05:28:36.783+0530] {logging_mixin.py:137} INFO - 2020-02-14 00:00:00
[2023-02-02T05:28:36.833+0530] {logging_mixin.py:137} INFO - 2020-02-15 00:00:00
[2023-02-02T05:28:39.922+0530] {logging_mixin.py:137} INFO - 2020-02-16 00:00:00
[2023-02-02T05:28:39.967+0530] {logging_mixin.py:137} INFO - 2020-02-17 00:00:00
[2023-02-02T05:28:40.120+0530] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/root/airflow/dags/NSE_Open_Interest.py", line 83, in extract
    dd,values = get_data(date)
TypeError: cannot unpack non-iterable int object
[2023-02-02T05:28:40.125+0530] {taskinstance.py:1322} INFO - Marking task as UP_FOR_RETRY. dag_id=NSE_Open_Interest, task_id=extract, execution_date=20230129T000000, start_date=20230201T235836, end_date=20230201T235840
[2023-02-02T05:28:40.146+0530] {standard_task_runner.py:100} ERROR - Failed to execute job 3578 for task extract (cannot unpack non-iterable int object; 6152)
[2023-02-02T05:28:40.180+0530] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-02-02T05:28:40.201+0530] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
